src/client/grid/move_intercept.tsx

```diff
@@ -74,123 +37,12 @@ export function useMoveInterceptionState() {
   };
 }
 
-const NO_ONE = "NO_ONE" as const;
-
-type SelectedPlayer = PlayerColor | typeof NO_ONE;
-
-export function InterceptMoveModal({
-  cityName,
-  moveData,
-  clearMoveData: clearMoveDataExternal,
-}: InterceptMoveModalProps) {
+export function InterceptMoveModal(props: InterceptMoveModalProps) {
   const gameKey = useGameKey();
```

Comment:
You should be able to use the method useViewSettings here.

Reply:
Oh, good call! I hadn't seen that helper yet.